### PR TITLE
master-lps-135362

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-135362
https://issues.liferay.com/browse/LPS-135369

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase [hunk 1]
```diff
@@ -80,11 +90,16 @@ definition {
...
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Content Page Name",
+			type = "content");
+
+		ContentPagesNavigator.openEditContentPage(
+			pageName = "Content Page Name",
+			siteName = "Test Site Name");
 
 		PageEditor.addFragment(
```
Comment (austinchiang) [inline modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase:104]: @liuxingye can you see if we can add fragments through JSON? Pretty sure there are existing macros here

File: modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase [hunk 2]
```diff
@@ -124,11 +137,16 @@ definition {
...
```
Comment (austinchiang) [inline modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase:151]: Same here as well as the other cases

File: modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase [hunk 3]
```diff
@@ -413,23 +415,24 @@ definition {
 
 		PageEditor.clickPublish();
 
-		User.logoutPG();
+		var portalURL = PropsUtil.get("portal.url");
```
Comment (austinchiang) [inline modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase:418]: This is ok for now but in the future we should try to put this into a macro. I've seen this a few times

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/SERPPreview.testcase [hunk 4]
```diff
@@ -62,105 +62,41 @@ definition {
...
 			locale = "zh-CN");
 
-		PortletEntry.inputText(
-			fieldLabel = "HTML Title",
-			text = "中文标题");
-
-		PortletEntry.inputText(
-			fieldLabel = "Description",
-			text = "中文描述");
+		SERPPreview.addGeneralSettings(
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/SERPPreview.testcase:97]: Nice

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**